packages/react/portal/src/Portal.stories.tsx

```diff
@@ -53,3 +54,74 @@ export const CustomContainer = () => {
     </>
   );
 };
+
+export const Chromatic = () => {
+  const portalContainerRef = React.useRef<HTMLDivElement>(null);
+  const portalContainerRef2 = React.useRef<HTMLDivElement>(null);
+
+  return (
+    <div style={{ padding: 150 }}>
+      <h1>Default (append to body)</h1>
+      <div style={{ padding: 10, margin: 10, border: '1px solid blue' }}>
+        <p>Container A</p>
+
+        <Portal>
+          <div style={{ padding: 10, margin: 10, border: '1px solid blue' }}>
+            <p>This content is rendered in a portal (another DOM tree)</p>
+            <p>
+              Because of the portal, it can appear in a different DOM tree from the main one (by
+              default a new element inside the body), even though it is part of the same React tree.
+            </p>
+          </div>
+        </Portal>
+      </div>
+
+      <h1>Custom container</h1>
+      <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+        <p>Container B</p>
+        <Portal containerRef={portalContainerRef}>
+          <div style={{ padding: 10, margin: 10, border: '1px solid green' }}>
+            <p>
+              This content is rendered in a portal inside Container B but appears inside Container B
+              because we have used Container C as a container element for the Portal.
```

Comment:
Should this say the following?

```diff
- This content is rendered in a portal inside Container B but appears inside Container B 
+ This content is rendered in a portal inside Container B but appears inside Container C 
```

That seems to be where it is appearing 🤔

Reply:
Yep, my bad, I changed that copy from the copy/paste of the old stories but forgot to update that one letter 😀